### PR TITLE
fix(build): android pkg name

### PR DESCRIPTION
**Pull request by ngjaying**

### CODE REVIEW

File: .github/workflows/build_packages.yaml [hunk 1]
```diff
@@ -123,9 +123,9 @@ jobs:
         run: |
           ver=$(git describe --tags --always)
           cd app
-          fyne package -os android/arm64 -name ekuiper -metadata version=$ver -release -appID github.com.lfedge.ekuiper -icon icon.png || true
+          fyne package -os android/arm64 -name kuiper-$ver -metadata version=$ver -release -appID github.com.lfedge.ekuiper -icon icon.png || true        
           mkdir -p ../_packages
-          mv ekuiper.apk ../_packages/
+          mv kuiper* ../_packages/
```
Comment (Rory-Z) [inline .github/workflows/build_packages.yaml:128]: We also need change 404 line

Comment (Rory-Z) [reply .github/workflows/build_packages.yaml:128]: I think we need to remind people in some way that the package name generated by fyne package is not `kuiper-$ver`, in order to reduce future maintenance costs.

Comment (ngjaying) [reply .github/workflows/build_packages.yaml:128]: How? Any suggestions?

Comment (Rory-Z) [reply .github/workflows/build_packages.yaml:128]: No good ideas, maybe we can add some comments.

**Review by Rory-Z — APPROVED** [review]

**Pull request merged**